src/js/options-page.js

```diff
@@ -125,20 +127,37 @@ function createPageWithPrefs(prefs) {
   }
   updatePageWithPrefs(prefs);
 }
-function exportHTML() {
+function exportStartTab() {
   const now = (new Date()).toISOString().slice(0, 10).replace(/-/g, '_');
-  const el = document.createElement('a');
-  el.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(OPTS.html));
-  el.setAttribute('download', `sst_backup_${now}.html`);
-  el.click();
-}
-function importHTML() {
+
+  const json = JSON.stringify(OPTS.json);
+  const blob = new Blob([json], { type: 'application/json' });
+  const url = URL.createObjectURL(blob);
+  const a = document.createElement('a');
+  a.href = url;
+  a.download = `sst_backup_${now}.json`;
+  a.click();
+  URL.revokeObjectURL(url);
+}
+function importStartTab() {
   util.simulateClick('#fileupload');
 }
 function importLoadedFile(file) {
   if (file.target && typeof file.target.result === 'string') {
-    OPTS.backup = OPTS.html;
-    OPTS.html = file.target.result;
+    OPTS.jsonBackup = [...OPTS.json];
+
+    // Keep support for old backups in HTML format
+    if (file.target.result.includes('<section')) {
+      OPTS.json = htmlStringToJson(file.target.result);
+    } else {
+      try {
+        OPTS.json = JSON.parse(file.target.result);
+      } catch (e) {
+        alert('Invalid file');
```

Comment:
Long term aim - a better (more graceful) error handler mechanism than an alert.
Short term - it's a linear process, so ok.